awx/main/tasks/system.py

```diff
@@ -508,7 +508,6 @@ def inspect_execution_nodes(instance_list):
 
 @task(queue=get_task_queuename, bind_kwargs=['dispatch_time', 'worker_tasks'])
 def cluster_node_heartbeat(dispatch_time=None, worker_tasks=None):
-    logger.debug("Cluster node heartbeat task.")
```

Comment:
The dispatcher task code logs when it's starting a task at DEBUG level, so this is pointless.